python_modules/dagster/dagster/_core/asset_graph_view/serializable_entity_subset.py

```diff
@@ -78,13 +107,14 @@ def try_from_coercible_value(
         key: T_EntityKey,
         value: CoercibleToAssetEntitySubsetValue,
         partitions_def: Optional[PartitionsDefinition],
+        dynamic_partitions_store: Optional[DynamicPartitionsStore],
```

Comment:
could make these pass a default None value, but tbh, the fact that the dynamic partition store param typically passes a None has bit me a couple times so i lean toward making it explicit that something needs to be passed

Reply:
or maybe since we only need the instance for dynamic partitions, and those have the default subset and won't be validated against the partition keys, i can just remove this entirely